modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/DiagramFooter.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayButton from '@clayui/button';
+import {ClaySelect} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import React from 'react';
+
+const DiagramFooter = ({
+	enableResetZoom,
+	selectedOption,
+	setChangedScale,
+	setResetZoom,
+	setSelectedOption,
+	setZoomInHandler,
+	setZoomOutHandler,
+	spritemap,
+}) => {
+	const handleZoomChange = (event) => {
+		setSelectedOption(event.target.value);
+		setChangedScale(true);
+	};
+
+	const options = [2, 1.75, 1.5, 1.25, 1, 0.75, 0.5];
```

Comment:
You can declare this constant outside the component's function and UPPERCASE it. Just to avoid this variable to being declared every time React call the component function :)

Reply:
:)